### ISSUE TITLE
[3.x] DocManager больше не может редактировать права доступа к ресурсам

### ISSUE THREAD

**Issue by sashabeep** [evolution-cms/evolution#2067]
Видимо, из-за изменений в базе

**Comment by leonie12** [comment]
After Google Translate I think that I saw this issue too. I wrote about it some days ago in the forums: „for example DocManager throws errors when trying to change permissions or for cloned pages it is impossible to change permissions – at least without working directly in the database“
I think here it is better placed than in the forums.

**Comment by sashabeep** [comment]
@leonie12 Yep, nobody patched DocManager for 3.x database modifications. "Documents Permissions" tab is unuseful for now
